WordPress/Classes/Utility/BackgroundTasks/WeeklyRoundupBackgroundTask.swift

```diff
@@ -32,7 +32,7 @@ private class WeeklyRoundupDataProvider {
     ///
     private let debugSettings = WeeklyRoundupDebugScreen.Settings()
 
-    init(coreDataStack: CoreDataStack, onError: @escaping (Error) -> Void) {
+    init(coreDataStack: CoreDataStackSwift, onError: @escaping (Error) -> Void) {
```

Comment:
@crazytonyli The type of `coreDataStack` should `ContextManager` or `CoreDataStackSwift`?

Reply:
We should use `CoreDataStackSwift` here: referencing the protocols instead of the concrete implementation `ContextManager`. The reason `CoreDataStackSwift` exists is I want to add Swift API to the `CoreDataStack` protocol, which is impossible because its is an Objective-C protocol. I ended up creating this new `CoreDataStackSwift` which inherits from `CoreDataStack`. So, for places where you want to use the Swift APIs (the ones that use Swift generic and Result type), you can reference `CoreDataStackSwift`.